engine/framework/internal/master/worker_manager.go

```diff
@@ -508,7 +508,7 @@ func (m *WorkerManager) checkWorkerEntriesOnce() error {
 			case frameModel.WorkerStatusFinished:
 				offlineError = derror.ErrWorkerFinish.FastGenByArgs()
 			case frameModel.WorkerStatusStopped:
```

Comment:
will we unify the `status` in other pr? Like:
`WorkerStatusStopped` -> `WorkerStatusCanceled`

Reply:
yes